src/index.js

```diff
@@ -0,0 +1 @@
+import './style.css';
```

Comment:
**Use ES6 modules to write modular JavaScript with proper ES6 syntax.**


- usage of `` arrow functions ``  instead of the `` function `` keyword.
- usage of `` import `` and `` export``  instead of require or `` module.exports``.
- proper usage of `` const`` and ``let`` keywords.